Babylon/utils/configuration.py

```diff
@@ -286,6 +285,28 @@ def get_platform_var(self, var_name) -> Optional[object]:
         else:
             return read_yaml_key(_path, var_name)
 
+    def set_deploy_var(self, var_name, var_value) -> None:
+        """
+        Set key value in current deployment configuration file
+        :param var_name: the key to set
+        :param var_value: the value to assign
+        """
+        if not (_path := self.get_deploy_path()).exists():
+            pass
+        else:
+            write_yaml_value(_path, var_name, var_value)
+
+    def set_platform_var(self, var_name, var_value) -> None:
+        """
+        Set key value in current platform configuration file
+        :param var_name: the key to set
+        :param var_value: the value to assign
+        """
+        if not (_path := self.get_platform_path()).exists():
+            pass
```

Comment:
same as last comment

Reply:
i use the returned value of get path in the line below